.github/workflows/unit-tests.yaml

```diff
@@ -15,7 +15,7 @@ on:
     types: [opened, edited, synchronize]
 jobs:
   run-unit-tests-amd64:
-    name: Run unit tests on amd64 for ${{ matrix.os }} - ${{ matrix.compiler }} - ${{ matrix.flb_option }}
+    name: amd64 for ${{ matrix.os }} - ${{ matrix.compiler }} - ${{ matrix.flb_option }}
```

Comment:
Personally I'd have the matrix stuff at the start then end with "unit tests on amd64".